packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -152,6 +153,26 @@ class AndroidValidator extends DoctorValidator {
       return ValidationResult(ValidationType.partial, messages, statusInfo: sdkVersionText);
     }
 
+    // Success.
+    return ValidationResult(ValidationType.installed, messages, statusInfo: sdkVersionText);
+  }
+
```

Comment:
Nit: nuke the extra newline here, and add one after the closing brace of the class.

Reply:
Done.